components/compliance-service/reporting/relaxting/util_test.go

```diff
@@ -1,23 +1,30 @@
 package relaxting
 
 import (
-	"github.com/chef/automate/components/compliance-service/ingest/ingestic/mappings"
-	"github.com/stretchr/testify/assert"
 	"testing"
 	"time"
+
+	"github.com/chef/automate/components/compliance-service/ingest/ingestic/mappings"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestGetEsIndexNoTimeLast24h(t *testing.T) {
 	sum := mappings.IndexNameSum
 	filters := make(map[string][]string)
 
-	index, err := GetEsIndex(filters, true)
-
-	today := time.Now().UTC().Format("2006.01.02")
-	yesterday := time.Now().Add(-24 * time.Hour).UTC().Format("2006.01.02")
+	todayTime := time.Now().UTC()
 
-	assert.Equal(t, sum+"-"+yesterday+"*,"+sum+"-"+today+"*", index)
+	index, err := GetEsIndex(filters, true)
 	assert.NoError(t, err)
+
+	today := todayTime.Format("2006.01.02")
```

Comment:
Can be avoided if we use `Day()` as mentioned below.

Reply:
This cannot be avoided as we use this format of date in else case to compare index string, but yes we can use Day()